src/BasePresenter.php

```diff
@@ -71,6 +71,18 @@ public function getRouteKeyName()
     }
 
     /**
+     * Retrieve model for route model binding.
+     *
+     * @param mixed $routeKey
+     *
+     * @return mixed
+     */
+    public function resolveRouteBinding($routeKey)
+    {
+        return $this->wrappedObject->resolveRouteBinding($routeKey);
```

Comment:
The method can be there, no-one will call that. See travis, 5.1-5.4 all passed. But I can create a version checking if around that.